client/src/components/categorical/category/index.tsx

```diff
@@ -485,7 +490,7 @@ const CategoryHeader = React.memo(
               hide: { enabled: false },
             }}
           >
-            <AnchorButton
+            <Button
```

Comment:
clean up - anchor button is an `<a>` that look like a button, but not a `<button>` tag. In this case, we want `<button`>